controllers/profile.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var bodyParser = require('body-parser');
+var session = require('express-session');
+var bcrypt = require('bcrypt');
+var db = require('../database.js');
+var config = require('../config.js');
+var MySQLStore = require('express-mysql-session')(session);
+var sessionStore = new MySQLStore(config.dbOptions);
+
+function requireAuthentication(req, res, next) {
+
+	if (!req.isAuthenticated()) {
+		return res.redirect('/login');
+	}
+
+	// The user is logged in.F
+	// Allow heir request to continue.
+	next();
+}
+
+
+module.exports = function(app) {
+
+	app.get('/profile', function(req, res, next) {
```

Comment:
This route should be protected by the `requireAuthentication` middleware from above, I think.